internal/executiondata.go

```diff
@@ -0,0 +1,25 @@
+/*
+ * Flow Emulator
+ *
+ * Copyright 2019 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package internal
+
+import (
+	"github.com/onflow/flow/protobuf/go/flow/executiondata"
+)
+
+type ExecutionDataAPIClient executiondata.ExecutionDataAPIClient
```

Comment:
a comment why is this needed

Reply:
it's actually not in this PR. it was needed for the tests that I left out